examples/i-inverse-kinematics.cpp

```diff
@@ -20,7 +20,7 @@ int main(int argc, char ** argv)
   pinocchio::Data::Matrix6x J(6,model.nv); J.setZero();
   unsigned int svdOptions = Eigen::ComputeThinU | Eigen::ComputeThinV;
 
-  for (int i=0 ; i<IT_MAX ; ++i)
+  for (int i=0 ; i<=IT_MAX ; ++i)
```

Comment:
Why this change? And what does your commit message mean?

Reply:
It is to address [this comment](https://github.com/stack-of-tasks/pinocchio/pull/673/files#r256545913).
I do not think it is the best solution, but I guess we can think about it in another moment.

Reply:
Yes, I took the quick & dirty path, sorry. This fixed an issue, but not with the best style. I think finding the best style can wait and be addressed outside of this PR.
@jcarpent please tell me if you want any change, or if we can resolve this conversation.

Reply:
It seems fine for now. Thanks to you.